main/src/ca/uwaterloo/flix/util/FileOps.scala

```diff
@@ -27,6 +28,24 @@ import scala.util.Using
 
 object FileOps {
 
+  /**
+    * Deletes `file` if it exists. Wraps any error `e` in `Result.Err(e)`.
```

Comment:
I guess it should say path, it can delete both files and directories, if we are trying to be precise.